CCL.Importer/PrefabWrangler.cs

```diff
@@ -768,7 +782,10 @@ private static void SetupBrakeGlows(GameObject newFab, Bogie front, Bogie rear,
             else
             {
                 // Or just use the same one as the base car type.
-                brakeGlow.overheatColor.colorGradient = baseLivery.prefab.GetComponent<BrakesOverheatingController>().overheatColor.colorGradient;
+                if (baseLivery.prefab.TryGetComponent(out BrakesOverheatingController baseGlow))
```

Comment:
It did not occur to me the caboose could be missing this component entirely since its brakes can't overheat. Should we provide a default gradient, possibly matching the usual vanilla one?
As it stands it might cause a wagon not to glow at all if the caboose is set as the base type.
Vanilla has an asset of type `BrakeShoeOverheatColorGradient`, which has only a gradient stored within, and it looks like the only one.
Possibly also make it the one that is applied by default to the custom brake glow component.